doc-content/enterprise-only/admin-and-config/managing-business-central-ssh-keys-con.adoc

```diff
@@ -0,0 +1,6 @@
+[id='managing-business-central-ssh-keys-con']
+= SSH Keys
+
+{Central} provides an SSH keystore service to enable user SSH authentication. It provides a configurable default SSH keystore, extensible APIs (for custom implementations), and support for multiple SSH public keys formats.
+
+You can access the *SSH Keys* option from the *Admin* page to enable users to register their SSH public keys.
```

Comment:
@domhanak I think that pars refers to the "admin page" so the bold shoud be *Admin page*. @gadeyredhat WDYT??

Reply:
@domhanak, yes, admin refers to the **Admin** page here. 
@pefernan, as per writing style guide only the name (a noun) of UI should be in bold and not the UI element. For example, say a screen has three tabs, Roles, Groups, & Users. Then only **Roles**, **Groups**, & **Users** should be in bold but not tab.

Reply:
> You can access the *SSH Keys* option from the *Admin* page to enable users to register their SSH public keys. 

The issue here is that we are saying: `you` (who? Admin?) can access `SSH Keys` option to enable `users` to register their SSH public keys. Imho this does not make sense because there is no option that enables SSH Keys usage for other users.

Reply:
Changed as suggested.